modules/postgresql/gcp/main.tf

```diff
@@ -27,6 +27,20 @@ resource "google_sql_database_instance" "instance" {
       }
     }
 
+    dynamic "database_flags" {
+      for_each = var.enable_detailed_logging ? [{
+        name  = "log_statement"
+        value = "all"
+        }, {
+        name  = "log_lock_waits"
+        value = "on"
+      }] : []
+      content {
+        name  = database_flags.value.name
+        value = database_flags.value.value
```

Comment:
What is the `database_flags` var here? Where is it coming from?

Reply:
It's the iteration value. It comes from the tag of the `dynamic` keyword.

Reply:
https://developer.hashicorp.com/terraform/language/expressions/dynamic-blocks